### PR TITLE
acala: update testnet contracts and VAA due to network reset

### PR DESCRIPTION
**Pull request by andreclaro**
acala: update testnet contracts and VAA due to network reset

### CODE REVIEW

File: deployments/testnet/tokenBridgeVAAs.csv [hunk 1]
```diff
@@ -10,7 +10,7 @@ Algorand (8) Testnet Token Bridge,010000000001006e90fe2a505f0d67b6ca6221a42f5595
...
-Acala (12) Testnet Token Bridge,01000000000100595f3b02b450ea71724c09a4b1a071f67eafb1471e0bd0d03d506e11ceedc81d788dd57dc81927cb74970ad6389e369d0589682d176eff033fe140dae1035155000000000100000001000100000000000000000000000000000000000000000000000000000000000000040000000002a49e7a00000000000000000000000000000000000000000000546f6b656e427269646765010000000c000000000000000000000000eba00cbe08992edd08ed7793e07ad6063c807004
+Acala (12) Testnet Token Bridge,01000000000100f14dcd0abe531c59fd5360750781a38c03fc1ba19b4fb39a4e91b2ee32dcaf5f6fad2dc1c83a7d20f7aa9fc8c57ad156aa29e1344bfc4f97123d30ebdea869eb010000000100000001000100000000000000000000000000000000000000000000000000000000000000040000000005c4aa5000000000000000000000000000000000000000000000546f6b656e427269646765010000000c00000000000000000000000064fb09e405d2043ed7785a29e296c766d56f2056
```
Comment (bruce-riley) [inline deployments/testnet/tokenBridgeVAAs.csv:13]: Wait, this looks like it's for the core address, not the token bridge?

Comment (andreclaro) [reply deployments/testnet/tokenBridgeVAAs.csv]: You are right! Updated the PR :) Thanks

Comment (andreclaro) [reply deployments/testnet/tokenBridgeVAAs.csv]:
```
worm generate registration --chain acala --module TokenBridge --contract-address 0xe157115ef34c93145Fec2FE53706846853B07F42 --guardian-secret SECRET

010000000001006b565b943c90f3634b8352f9089e3b3a0a50e6f32edea47b91b5a1c15f190a0317a7ffc9fc76df4ed98eb25effe70c0218bdcb16ff9799d1ed8b33e756ca08990100000001000000010001000000000000000000000000000000000000000000000000000000000000000400000000059f9b1700000000000000000000000000000000000000000000546f6b656e427269646765010000000c000000000000000000000000e157115ef34c93145fec2fe53706846853b07f42
```

**Review by evan-gray — APPROVED** [review]

**Review by SEJeff — APPROVED** [review]
Nothing like a bit of removal of cruft along with a chain reset.

**Pull request merged**